shared/actions/login.js

```diff
@@ -310,19 +310,9 @@ const displayAndPromptSecretSaga = onBackSaga =>
 
 const promptNewDeviceNameSaga = onBackSaga =>
   function*({existingDevices, errorMessage}) {
-    yield Saga.put(SignupGen.createSetDeviceNameError({deviceNameError: errorMessage}))
+    yield Saga.put(SignupGen.createSubmitDevicenameDoneError({devicename: '', error: errorMessage}))
```

Comment:
this reuses the signup store which is kinda bad but i don't want to touch login now